command/macro/verify/verify.go

```diff
@@ -0,0 +1,127 @@
+package verify
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	bd "github.com/deepsourcelabs/cli/analyzers/build"
+	vl "github.com/deepsourcelabs/cli/analyzers/validator"
+	"github.com/deepsourcelabs/cli/utils"
+	"github.com/spf13/cobra"
+)
+
+var (
+	configFolder     string
+	analyzerTOMLPath string
+	issuesDirPath    string
+)
+
+type MacroVerifyOpts struct{}
+
+func NewCmdMacroVerify() *cobra.Command {
+	// Configuring the default values for analyzer.toml and issues directory
+	cwd, _ := os.Getwd()
+	configFolder = filepath.Join(cwd, ".deepsource/analyzer")
```

Comment:
Look for `.deepsource` directory in parent directories also, if not found in cwd.